workbook/04-contributor.asciidoc

```diff
@@ -0,0 +1,178 @@
+
+Why 3 is incorrect: Your work on another team’s project reflects your own team’s needs, but you are not working for the other team. If the hosting team (or any community participant) is in a bind, you and your management might offer help as a gesture to benefit the whole company., However, it is not your task to make up for the time or staffing shortcomings at the hosting team.
+
+Why 4 is incorrect: Working on other features of a project, or doing background work such as reorganizing code or documentation, may indirectly contribute to your team’s success, so it is legitimate for you to invest time on those things.
+ 
+
+=== SEGMENT: The Contributor Ethos
+
+TIP:
+More than one answer may be correct in some questions.
+
+==== Question 1.  Which approach can drive success in your contribution to an InnerSource project?
+
+. My team’s work is crucial for the company’s success, so my team’s voice has more weight in a shared solution community than others.
+. As I a guest to the project, I am entitled to quick and comprehensive answers to my questions from host team.
+. As a guest to the project, I should make sure I understand and adhere to the rules and guidelines as outlined in the associated documentation (starting with, but not limited to, the README.md and CONTRIBUTING.md files) and can ask questions respectfully afterward for clarifications or help.
+. My commitment is to the change I provide. Once my contribution is accepted my work is done and I can focus on the core of my own project again.
+
+===== Correct answer:  3
+
+Why 1 is incorrect: Even if your team may play a more central role in the company, you are still the guest when interacting with an InnerSource project.  Ultimate accountability for decisions made about the project lies with its trusted committers.
+
+Why 2 is incorrect: There is nothing wrong with asking good questions, and they should be responded to in a timely fashion. However, you are a guest and need to respect the time and effort of others in the community. Therefore, make sure to read and understand all available documentation before asking questions, and be prepared for answers from any educated project member, not just host team leaders. This helps your status in the community and avoids randomization.
+
+Why 3 is correct: InnerSource stresses the creation of documentation, both as background to the work (for instance, README.md and CONTRIBUTING.md, and to justify individual decisions and code changes. The reason for a culture of documentation is to provide you, as a contributor, with the background you need to fit your change successfully into the project.
+
+Why 4 is incorrect: Getting your contribution accepted is an achievement to be celebrated. However, your involvement does not end there. You should plan to be available at a minimum to fulfill a 30-day warranty on your changes (and their impact), or even better: stay close to the community and help out with additional contributions. 
```

Comment:
Maybe add a link to https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/30-day-warranty.md

Reply:
Thought so too once creating the pull request. Only felt like a quick ```sed```&go though at that point.

```suggestion
Why 4 is incorrect: Getting your contribution accepted is an achievement to be celebrated.
However, your involvement does not end there. 
You should plan to be available at a minimum to fulfil a [30-day warranty](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/30-day-warranty.md) on your changes (and their impact), or even better: stay close to the community and help out with additional contributions. 
```